katas/content/deutsch_algo/implement_algo/solution.md

```diff
@@ -4,8 +4,8 @@ Follow the algorithm as outlined in the previous section:
 2. Apply the $H$ gate to the qubit.
 3. Apply the oracle. The syntax for applying the oracle is the same as for applying any other gate or operation.
 4. Apply the $H$ gate to the qubit again.
-5. Measure the qubit: if the measurement result is `Zero`, the function is constant, otherwise it is variable.
-Since we need to return `true` if the function is constant, we can just return the result of comparing the measurement result with `Zero`.
+5. Measure the qubit: if the measurement result is `Zero`, the function is constant, otherwise it's variable.
```

Comment:
I see a lot of shortening in these PRs (it is -> it's, etc.) Is this considered a better style?

Reply:
Not universally :-) Technical books and other formal writing scenarios discourage use of contractions. @SoniaLopezBravo  I assume there is a guideline somewhere about Microsoft position on contractions in learning materials?

Reply:
Yes, using contractions is part of the [Microsoft Writing Style Guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36389). There are many other guidelines to help create a casual tone.